templates/front-deployment.yaml

```diff
@@ -24,11 +24,19 @@ spec:
       containers:
         - env:
             - name: API_URL
-              value: {{ .Values.apiUrl | quote }}
+              value: {{- if .Values.LAGO_DOMAIN }}
```

Comment:
this is already managed by FE : https://github.com/getlago/lago-front/commit/f74e6e539f13d5227dcdff38eca03e25e33e9352

We just need to add the `LAGO_DOMAIN`  env var here!